GCRCatalogs/register.py

```diff
@@ -680,6 +714,5 @@ def retrieve_paths(name_startswith=None, name_contains=None, **kwargs):
     The format would be [(catalog_name, original_path, resolved_path), ...]
     """
     return _config_register.retrieve_paths(name_startswith=name_startswith, name_contains=name_contains, **kwargs)
-
-
 _config_register = ConfigRegister(_CONFIG_DIRPATH, _SITE_CONFIG_PATH)
+
```

Comment:
Two blank lines after module-level function definition. 
Leave only one blank line at the end of file (currently there's two; GitHub hides the last one automatically.)

```suggestion


_config_register = ConfigRegister(_CONFIG_DIRPATH, _SITE_CONFIG_PATH)
```